cvat/apps/lambda_manager/views.py

```diff
@@ -380,7 +383,17 @@ def reset(self):
 
             for anno in annotations:
                 label_id = labels.get(anno["label"])
-                if label_id is not None:
+                if label_id is None:
+                    continue # Invalid label provided
+                if anno["type"].lower() == "tag":
```

Comment:
Could you please contribute any serverless function for classification? Thus we can test the PR.

Reply:
I don't think the introduction of a new serverless function is appropriately within scope for #3358

I actually don't even have one I can share: the code I have that generates full-frame detections is closed source.  I can provide some instructions about how to temporarily modify an existing function, but I don't really have time to write a new one.  I'm not really an ML developer.

The problem here seems to be one of documentation: the only way to understand the API between CVAT and Nuclio functions is to read this `views.py` file, so I think documentation would be ideal.

Reply:
@subdavis , it will be great to hear how in your company (kitware?) you are using CVAT to annotate data.

Reply:
@subdavis , thanks for sharing the information.